src/components/CurriculumDownloadButtons/CurriculumDownloadButton.tsx

```diff
@@ -71,9 +71,9 @@ const CurriculumDownloadButton: FC<CurriculumDownloadProps> = ({
   };
 
   return (
-    <Flex>
-      {lessonPage && keyStageSlug === "ks4" && subjectSlug === "maths" ? (
-        <Flex $flexDirection={"column"}>
+    <>
+      <Flex>
+        {!tier && keyStageSlug === "ks4" && subjectSlug === "maths" ? (
```

Comment:
I know this isn't new but it would be helpful to pull this kind of long conditional out into a named variable for better readability - something like `isKs4MathsWithoutTier`? Even better if it could explain why we're doing this (as I'm not really sure hah)

Reply:
Yeah good point, have added to a variable and left a comment.

If we are on a Maths programme page, we want to download a zip of foundation, core, higher curriculum map .pdfs. This is pretty brittle atm but will be removed shortly after the launch as curriculums maps are updated.